README.md

```diff
@@ -1,8 +1,130 @@
 # cozy-client
+
 A simple and declarative way of managing [cozy-stack](https://github.com/cozy/cozy-stack) API calls and resulting data.
 
 ## Introduction
+
 `cozy-client` is a convenient yet powerful way to bind `cozy-stack` queries to your (p)React components.
 
 If you don't use React, you can still benefit of its underlying libraries:
- - [cozy-stack-link](https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-link/README.md): a HTTP client to the cozy stack.
\ No newline at end of file
+
+* [cozy-stack-link](https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-link/README.md): a HTTP client to the cozy stack.
+
+## Setup
+
+### Install
+
+`npm install --save cozy-client cozy-stack-link`
+or
+`yarn add cozy-client cozy-stack-link`
+
+To get started using `cozy-client` with (p)React, you need to create a `CozyClient`, a `CozyStackLink`, and a `CozyProvider`:
+
+* `CozyClient` is the master of your data: it manages data queries and their status ;
+* `CozyStackLink` is the HTTP interface to the stack instance ;
+* `CozyProvider` injects the client into components' context.
+
+#### Creating a client
+
+```js
+import CozyClient from 'cozy-client'
+import CozyStackLink from 'cozy-stack-link'
+
+// We need to retrieve the domain name of the cozy instance and a token from the HTML
+// See https://cozy.github.io/cozy-docs-v3/en/dev/app/#behind-the-magic
+const root = document.querySelector('[role=application]')
+const data = root.dataset
+
+const client = new CozyClient({
+  link: new CozyStackLink({
+    uri: `${window.location.protocol}//${data.cozyDomain}`,
+    token: data.cozyToken
+  })
+})
+```
+
+#### Creating a provider
+
+To connect the client to your component tree, use a `CozyProvider` somewhere high in your app:
+
+```js
+import CozyClient, { CozyProvider } from 'cozy-client'
+
+const client = new CozyClient({
+  /*...*/
+})
+
+ReactDOM.render(
+  <CozyProvider client={client}>
+    <MyApp />
+  </CozyProvider>,
+  document.getElementById('main')
+)
+```
+
```

Comment:
Could you add a small sentence or two regarding what provides the `CozyProvider`?